src/test-utils/dom/segmented-control/index.ts

```diff
@@ -13,4 +19,13 @@ export default class SegmentedControlWrapper extends ComponentWrapper {
   findSelectedSegment(): ElementWrapper | null {
     return this.findByClassName(styles.selected);
   }
+
+  /**
+   * Finds the segment with the given ID and returns the element.
```

Comment:
Nitpick, non-blocking but I find "the element" a bit unnecessary. What about:

```Finds the segment with the given ID and returns it.```

or

```Returns the segment with the given ID.```

or just

```Finds the segment with the given ID.```